bitcash/network/services.py

```diff
@@ -378,16 +195,16 @@ def broadcast_tx(cls, tx_hex, network="mainnet"):  # pragma: no cover
         """
         success = None
 
-        for api_call in cls.BROADCAST_TX:
+        for endpoint in get_endpoints_for(network):
             try:
-                success = api_call(tx_hex, network)
+                success = endpoint.broadcast_tx(tx_hex, timeout=DEFAULT_TIMEOUT)
                 if not success:
                     continue
                 return
             except cls.IGNORED_ERRORS:
                 pass
 
-        if success is False:
+        if not success:
```

Comment:
I  think this line went backwards. I would do `if success is False` or `if success is not True`

Reply:
Hmm, as plain English I felt that "if not success" reads more naturally than "if success is False" .. but if it's preferred to keep it that way I don't have any particular objections, just trying to improve readability where I can.

Reply:
I'm pretty sure flake8 will complain here. From what I understand, the Pythonic way is generally `if var is bool:`.

Reply:
Ok, flake8 doesn't seem to mind.